netket/models/mlp.py

```diff
@@ -59,7 +59,7 @@ class MLP(nn.Module):
     If None is provided, the output layer will be essentially linear."""
     use_hidden_bias: bool = True
     """if True uses a bias in the hidden layer."""
-    use_output_bias: bool = False
```

Comment:
Consistency with  `nk.nn.blocks.MLP`?

I am potentially against this change because it's a breaking change of a model, against there is a strong motivation.

Though 99% of the time one would use this as `True`, right?

Reply:
The reasoning at the time was that log cte only affects the scale of the model. But it's such a small detail and I personally use this module as a block elsewhere.

Reply:
It doesn't break anything I'd say..

Reply:
Ok, I changed it back. The other fixes are more pressing.